src/EditorFeatures/CSharpTest/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -196,86 +195,91 @@ int M(int i)
                 break;
         }
     }
-}");
+}";
+
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
         public async Task TestMissingOnDefaultBreak_01()
         {
-            await TestMissingAsync(
-@"class Program
+            var code = @"class Program
 {
     void M(int i)
     {
-        [||]switch (i)
+        switch (i)
         {
             default:
                 break;
         }
     }
-}");
+}";
+
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
         public async Task TestMissingOnDefaultBreak_02()
         {
-            await TestMissingAsync(
-@"class Program
+            var code = @"class Program
 {
     void M(int i)
     {
-        [||]switch (i)
+        switch (i)
         {
-            case _:
+            case { }:
```

Comment:
📝 This test arguably was never correct and likely could have been removed. However, based on context (other tests with related names) I decided to use another form in the spirit of the original.

Reply:
fwiw, I think it should have been `var _` - `case {}` doesn't behave like `default:` (tests for null)

Reply:
`case var _` is handled by `TestMissingOnDefaultBreak_03` below

Reply:
Also note that `i` is a non-nullable value type, so `{ }` and `var _` are equivalent.